Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -196,62 +196,80 @@ internal async ValueTask<List<Result>> RequestUpdateAsync(string search, Cancell
-                                    Log.Exception("PluginsManager", $"Update failed for {x.Name}",
-                                        t.Exception.InnerException, "RequestUpdate");
-                                    Context.API.ShowMsg(
-                                        Context.API.GetTranslation("plugin_pluginsmanager_install_error_title"),
-                                        string.Format(
-                                            Context.API.GetTranslation("plugin_pluginsmanager_install_error_subtitle"),
-                                            x.Name));
-                                }, TaskContinuationOptions.OnlyOnFaulted);
+                                        Context.API.RestartApp();
+                                    }).ContinueWith(t =>
+                                    {
+                                        Log.Exception("PluginsManager", $"Update failed for {x.Name}",
+                                            t.Exception.InnerException, "RequestUpdate");
+                                        Context.API.ShowMsg(
+                                            Context.API.GetTranslation("plugin_pluginsmanager_install_error_title"),
+                                            string.Format(
+                                                Context.API.GetTranslation("plugin_pluginsmanager_install_error_subtitle"),
+                                                x.Name));
+                                    }, TaskContinuationOptions.OnlyOnFaulted);
+
+                                    return true;
+                                }
 
-                                return true;
-                            }
+                                return false;
+                            },
+                            ContextData =
+                                new UserPlugin
+                                {
+                                    Website = x.PluginNewUserPlugin.Website,
+                                    UrlSourceCode = x.PluginNewUserPlugin.UrlSourceCode
+                                }
+                        });
 
-                            return false;
-                        },
-                        ContextData =
-                            new UserPlugin
-                            {
-                                Website = x.PluginNewUserPlugin.Website, UrlSourceCode = x.PluginNewUserPlugin.UrlSourceCode
-                            }
-                    });
+            if (resultsForUpdate.Count() > 1)
+            {
+                var updateAllResult = new Result
+                {
+                    Title = Context.API.GetTranslation("plugin_pluginsmanager_update_all_title"),
+                    SubTitle = Context.API.GetTranslation("plugin_pluginsmanager_update_all_subtitle"),
+                    IcoPath = icoPath,
+                    Action = e =>
+                    {
+                        // TODO: logic here
```

Comment:
Please remove :)

Reply:
I think he means to remove the `TODO` comment once you finish it.

Reply:
Yeah, please remove the todo comment or do the todo please.